File: MonkeyCoreVB/Controls/ListBox_NoFlicker.vb
﻿Imports System.Windows.Forms

Namespace Controls

    Public Class ListBox_NoFlicker

        Inherits ListBox

#Region "Public Constructors"

        Public Sub New()
            MyBase.New()
            Me.DoubleBuffered = True
        End Sub

#End Region

    End Class

End Namespace

File: MonkeyCore.WinFormsControls/ListBox_NoFlicker.cs
using System.Windows.Forms;

namespace Controls
{
    /// <summary>
    /// Double Buffered ListBox
    /// </summary>
    /// <seealso cref="System.Windows.Forms.ListBox" />
    public class ListBox_NoFlicker : ListBox
    {
        #region Public Constructors

        /// <summary>
        /// Initializes a new instance of the <see cref="ListBox_NoFlicker"/> class.
        /// </summary>
        public ListBox_NoFlicker() : base()
        {
            this.DoubleBuffered = true;
        }

        #endregion Public Constructors
    }
}